### PR TITLE
[docs] How to use in-memory DB for tests data

### PR DESCRIPTION
**Pull request by tshkursk**

### CODE REVIEW

File: docs/modules/guides/pages/create-and-connect-h2-db.adoc [hunk 1]
```diff
@@ -0,0 +1,41 @@
+= How to use H2 database in an embedded mode (in memory)
+
+Assuming there is a project containing VIVIDUS-based tests
+(e.g. https://github.com/vividus-framework/vividus-starter[VIVIDUS Starter])
+and the goal is to create http://h2database.com/html/features.html#feature_list[H2 database]
```
Comment (ikalinin1) [inline docs/modules/guides/pages/create-and-connect-h2-db.adoc:5]: http => https

**Comment by ikalinin1** [comment]
I think it makes sense to add an example with the transformer

**Comment by tshkursk** [comment]
> I think it makes sense to add an example with the transformer

Looks like transformer won't be merged soon. So not actual for this PR for now.

File: docs/modules/guides/pages/use-in-memory-db-to-manage-test-data.adoc [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+https://h2database.com/html/features.html#feature_list[H2 database]).
+The database is deployed on tests start, accessible from tests and shutdown on
+tests end. The test data can be created from SQL files located in the project.
+
+. Add the required dependencies (DB plugin, H2 database) to the project `build.gradle`
++
+[source,gradle,subs="attributes+"]
+----
+implementation(group: 'org.vividus', name: 'vividus-plugin-db', version: '{current-version}')
+implementation(group: 'com.h2database', name: 'h2', version: '1.4.200')
```
Comment (uarlouski) [inline docs/modules/guides/pages/use-in-memory-db-to-manage-test-data.adoc:15]: I think that by using specific version pf h2 we put updating responsibility on our shoulders

File: docs/modules/guides/pages/use-in-memory-db-to-manage-test-data.adoc [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+. Configure connection to the H2 database
++
+[source,properties]
+----
+db.connection.testh2db.driver-class-name=org.h2.Driver
+db.connection.testh2db.url=jdbc:h2:mem:test;\
+  INIT=create schema if not exists test\\;\
+  SET SCHEMA test\\;\
+  RUNSCRIPT FROM 'classpath:/path/to/sql/file1.sql'\\;\
```
Comment (ikalinin1) [inline docs/modules/guides/pages/use-in-memory-db-to-manage-test-data.adoc:26]: I think it makes sense to describe what is classpath:/ prefix is and how it maps to the regular VIVIDUS test project

File: docs/modules/guides/pages/use-in-memory-db-to-manage-test-data.adoc (hunk 2 above)
Comment (tshkursk) [reply docs/modules/guides/pages/use-in-memory-db-to-manage-test-data.adoc:15]: Was done similar to this: https://docs.vividus.dev/vividus/latest/guides/execute-sql-against-csv.html
Specified version works as described in this guide. The others - we can not be sure.

**Review by ikalinin1 — APPROVED** [review]

File: docs/modules/guides/pages/use-in-memory-db-to-manage-test-data.adoc (hunk 2 above)
Comment (valfirst) [reply docs/modules/guides/pages/use-in-memory-db-to-manage-test-data.adoc:15]: we'll keep explicit version in a sakeness of simplicity

**Review by uarlouski — APPROVED** [review]

**Review by valfirst — APPROVED** [review]

**Pull request merged**